instrumentation/opentelemetry-instrumentation-starlette/src/opentelemetry/instrumentation/starlette/__init__.py

```diff
@@ -186,20 +205,46 @@ def _instrument(self, **kwargs):
         _InstrumentedStarlette._client_response_hook = kwargs.get(
             "client_response_hook"
         )
+        _InstrumentedStarlette._meter_provider = kwargs.get("_meter_provider")
+
         applications.Starlette = _InstrumentedStarlette
 
+        for instance in _InstrumentedStarlette._instrumented_starlette_apps:
+            self.instrument_app(
+                instance,
+                server_request_hook=_InstrumentedStarlette._server_request_hook,
+                client_request_hook=_InstrumentedStarlette._client_request_hook,
+                client_response_hook=_InstrumentedStarlette._client_response_hook,
+                tracer_provider=_InstrumentedStarlette._tracer_provider,
+            )
```

Comment:
Pass in `meter_provider`?

Reply:
sure will add this too. Forgot to add it